Common/util/memorystream.cpp

```diff
@@ -130,7 +130,7 @@ bool MemoryStream::Seek(soff_t offset, StreamSeek origin)
         return false;
     }
     _pos = static_cast<size_t>(std::max<soff_t>(0, pos));
-    _pos = static_cast<size_t>(std::min<soff_t>(_len, pos)); // clamp to EOS
+    _pos = static_cast<size_t>(std::min<soff_t>(_len, _pos)); // clamp to EOS
```

Comment:
Since both `_len` and `_pos` are `size_t` here, specifying std::min as `soff_t` is no longer right, and casting to `size_t` is also not necessary.

Reply:
removed the specifier and the casting!

https://github.com/adventuregamestudio/ags/compare/bac760d52b4ffddc7c410c73c04795821272d5cb..1c55df6a73bc1a1528c7f9b95304cb382ea11c3a